client/src/Components/MetaViewer/MetaViewer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export interface MetaViewerProps {
+    "datatype": string;
+    "version": string;
+    "offset": number;
+    "sample_rate": number;
+    "description": string;
+}
+
+export const MetaViewer: React.FC<MetaViewerProps> = (meta: MetaViewerProps) => {
+    const truncateDescription = (desc: string) => {
+        return desc.length > 50 ? desc.substring(0, 50) + "..." : desc;
+    }
+    return ( 
+        <div className="border border-primary ml-3 mt-2">
+            <div className="flex mt-3 mb-3 stats shadow">
+            <div className="stat place-items-center">
```

Comment:
Small thing, is the alignment of these divs right? Should it be indented one more?

Reply:
it seems to be for me